modules/hcp-dc-image-mirror.adoc

```diff
@@ -68,7 +68,7 @@ mirror:
 ----
 +
 <1> Replace `<dns.base.domain.name>` with the DNS base domain name.
-<2> Replace `4.x.y-build` with the supported {product-title} version you want to use.
+<2> Replace `<4.x.y-build>` with the supported {product-title} version you want to use.
```

Comment:
In the YAML example, be sure to put angle brackets around each instance of `4.x.y-build` to match the text in this annotation.

Reply:
ah totally missed on  those ones. Updated all places.